test/function/entry.cpp

```diff
@@ -548,20 +548,20 @@ namespace {
   }
 
   TEST_F( EntryTest, SetAppName ) {
```

Comment:
it appears that this function had its indentation inadvertently changed - be sure to revert it (as no changes were made)